README.md

```diff
@@ -21,7 +21,7 @@ Maven:
   <dependency>
 	  <groupId>com.midtrans</groupId>
 	  <artifactId>java-library</artifactId>
-	  <version>2.1.1</version>
+	  <version>3.1.1</version>
```

Comment:
When working with `<major>.<minor>.<patch>` versioning format, if the major is increased, all the lower level should reset to 0. So this should be version `3.0.0`